.travis.yml

```diff
@@ -0,0 +1,26 @@
+dist: trusty
+sudo: false
+services:
+  - postgresql
+addons:
+  postgresql: "9.6"
+language: python
+python:
+  - "3.4"
+  - "3.5"
+  - "3.6"
+install:
+  - pip install .
+  - pip install -r serveradmin/requirements.txt
```

Comment:
Shouldn't the mail project be installed later?

Reply:
Makes more sense, yeah.